services/asset/properties/schemas.go

```diff
@@ -202,36 +131,111 @@ var propertySchema = map[string]*schema.Schema{
 		ValidateFunc: validation.StringInSlice(validPropertyTypes, false),
 		Description:  helper.AllowedValuesToDescription(validPropertyTypes),
 	},
+	"built_in": {
+		Type:        schema.TypeBool,
+		Computed:    true,
+		Description: "Indicates if it is a build-in property.",
+	},
+	"additional_properties": {
```

Comment:
Apparently its purpose is to add new properties in case of an unexpected event. Maybe we shouldn't expose this field in terraform ? What do you think ?

Reply:
I don't see it in the API, if not then it shouldn't be exposed

Reply:
Yes I can't see it either on OpenAPI, it is only visible in the backend code. I will delete this.